default_settings.json

```diff
@@ -0,0 +1,5254 @@
+	"editor.gotoLocation.alternativeDefinitionCommand": "editor.action.goToReferences",
+
+	// Alternative command id that is being executed when the result of 'Go to Implementation' is the current location.
+	"editor.gotoLocation.alternativeImplementationCommand": "",
+
+	// Alternative command id that is being executed when the result of 'Go to Reference' is the current location.
+	"editor.gotoLocation.alternativeReferenceCommand": "",
+
+	// Alternative command id that is being executed when the result of 'Go to Type Definition' is the current location.
+	"editor.gotoLocation.alternativeTypeDefinitionCommand": "editor.action.goToReferences",
+
+	// This setting is deprecated, please use separate settings like 'editor.editor.gotoLocation.multipleDefinitions' or 'editor.editor.gotoLocation.multipleImplementations' instead.
+	// 
+	"editor.gotoLocation.multiple": null,
+
+	// Controls the behavior the 'Go to Declaration'-command when multiple target locations exist.
+	//  - peek: Show peek view of the results (default)
+	//  - gotoAndPeek: Go to the primary result and show a peek view
+	//  - goto: Go to the primary result and enable peek-less navigation to others
+	"editor.gotoLocation.multipleDeclarations": "peek",
+
+	// Controls the behavior the 'Go to Definition'-command when multiple target locations exist.
+	//  - peek: Show peek view of the results (default)
+	//  - gotoAndPeek: Go to the primary result and show a peek view
+	//  - goto: Go to the primary result and enable peek-less navigation to others
+	"editor.gotoLocation.multipleDefinitions": "peek",
+
+	// Controls the behavior the 'Go to Implementations'-command when multiple target locations exist.
+	//  - peek: Show peek view of the results (default)
+	//  - gotoAndPeek: Go to the primary result and show a peek view
+	//  - goto: Go to the primary result and enable peek-less navigation to others
+	"editor.gotoLocation.multipleImplementations": "peek",
+
+	// Controls the behavior the 'Go to References'-command when multiple target locations exist.
+	//  - peek: Show peek view of the results (default)
+	//  - gotoAndPeek: Go to the primary result and show a peek view
+	//  - goto: Go to the primary result and enable peek-less navigation to others
+	"editor.gotoLocation.multipleReferences": "peek",
+
+	// Controls the behavior the 'Go to Type Definition'-command when multiple target locations exist.
+	//  - peek: Show peek view of the results (default)
+	//  - gotoAndPeek: Go to the primary result and show a peek view
+	//  - goto: Go to the primary result and enable peek-less navigation to others
+	"editor.gotoLocation.multipleTypeDefinitions": "peek",
+
+	// Controls whether bracket pair guides are enabled or not.
+	//  - true: Enables bracket pair guides.
+	//  - active: Enables bracket pair guides only for the active bracket pair.
+	//  - false: Disables bracket pair guides.
+	"editor.guides.bracketPairs": false,
```

Comment:
I think this should be `true`.

<img width="179" alt="Screen Shot 2022-04-22 at 3 16 46 PM" src="https://user-images.githubusercontent.com/811217/164787706-60350f38-0c57-458a-a618-f7e41ef2772

<img width="179" alt="Screen Shot 2022-04-22 at 3 16 46 PM" src="https://user-images.githubusercontent.com/811217/164787768-493985ad-8f3f-453e-9c52-502ebac4cbcd.png">
7.png">